devenv/lib/context.py

```diff
@@ -0,0 +1,14 @@
+from __future__ import annotations
+
+from typing import TypeAlias
+from typing import TypedDict
+
+from devenv.lib.repository import Repository
+
+Path: TypeAlias = str
```

Comment:
I avoid type aliases -- especially ones line this since they don't actually increase type safety here (but create indirection)

this is especially confusing because there is a common `Path` type from pathlib

but also type aliases are transitive so they don't help with enforcing stricter typing:

```python
Path: TypeAlias = str
y = 'foo'
reveal_type(y)  # str
x: Path = y  # not an error
```

there's `NewType` -- but I haven't found a usecase beyond opaque credential typing that isn't a huge pain without much gain

Reply:
Yeah, this didn't provide any level of edit-time insight that I was hoping for. Will remove after I get all feedback.